java/runtime/src/main/java/io/ray/runtime/functionmanager/FunctionManager.java

```diff
@@ -204,12 +205,28 @@ RayFunction getFunction(JavaFunctionDescriptor descriptor) {
       Map<Pair<String, String>, RayFunction> map = new HashMap<>();
       try {
         Class clazz = Class.forName(className, true, classLoader);
-
         List<Executable> executables = new ArrayList<>();
         executables.addAll(Arrays.asList(clazz.getDeclaredMethods()));
-        executables.addAll(Arrays.asList(clazz.getConstructors()));
+        executables.addAll(Arrays.asList(clazz.getDeclaredConstructors()));
+
+        Class clz = clazz;
+        clz = clz.getSuperclass();
+        while (clz != null && clz != Object.class) {
+          executables.addAll(Arrays.asList(clz.getDeclaredMethods()));
+          clz = clz.getSuperclass();
+        }
+
+        // Put interface methods ahead, so that in can be override by subclass methods in `map.put`
```

Comment:
Add a test case for interface methods with default implementation.

Reply:
added